src/components/Pages/PlaylistCollection/Container.js

```diff
@@ -66,6 +56,24 @@ const PlaylistCollectionContainer = ({ cId, t, cuId }) => {
   }, [cId, dispatch, collection, wipMap.collections]);
 
   useEffect(() => {
+    const fetchWindow = () => {
```

Comment:
@wissog you create functions inside of effect?

Reply:
you have to if you use that function inside the effect